mbof/views.py

```diff
@@ -1,21 +1,23 @@
 import os
+import datetime
```

Comment:
Is this import necessary?   Try removing it and see if the app runs without trouble.  The `timezone` object used later in this module is provided by `django.utils`, so you might not need to import `datetime` here.